docs/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/rollbacks.md

```diff
@@ -1,6 +1,31 @@
 ---
 title: Rollbacks
 ---
+## Additional Steps for Rollbacks with Rancher v2.6.4+
+
+Rancher v2.6.4 upgrades the cluster-api module from v0.4.4 to v1.0.2. Version v1.0.2 of the cluster-api, in turn, upgrades the CAPI CRDs apiVersion from `cluster.x-k8s.io/v1alpha4` to `cluster.x-k8s.io/v1beta1`. The CAPI CRDs upgrade to v1beta1 causes rollbacks to fail when you attempt to move from Rancher v2.6.4 to any previous version of Rancher v2.6.x. This is because v1beta1 is missing the previous versions that the CRDs need to roll back.
```

Comment:
```suggestion
Rancher v2.6.4 upgrades the cluster-api module from v0.4.4 to v1.0.2. Version v1.0.2 of the cluster-api, in turn, upgrades the Cluster API's  Custom Resource Definitions (CRDs) from `cluster.x-k8s.io/v1alpha4` to `cluster.x-k8s.io/v1beta1`. The CRDs upgrade to v1beta1 causes rollbacks to fail when you attempt to move from Rancher v2.6.4 to any previous version of Rancher v2.6.x. This is because necessary components in v1alpha4 are no longer available in v1beta1.
```